Python/compile.c

```diff
@@ -62,10 +62,12 @@
 #define SUCCESS 0
 #define ERROR -1
 
-#define RETURN_IF_ERROR(X)  \
-    if ((X) == -1) {        \
-        return ERROR;       \
-    }
+#define RETURN_IF_ERROR(X) \
+    do { \
+        if ((X) == -1) {        \
+            return ERROR;       \
+        } \
+    } while (0)
```

Comment:
```suggestion
#define RETURN_IF_ERROR(X) do { \
      if ((X) == -1) {        \
          return ERROR;       \
      } \
} while (0)
```

you can try this style, so the diff would be smaller.

Reply:
See: https://github.com/python/cpython/pull/119982#discussion_r1624616907

Reply:
Apart from that, I've actually spotted an inconsistency where I put a space before "(0)" (I've seen both `} while(0)` and `} while (0)`, but since I've started using `} while(0)`, I'll stick to it).